dataprovider/src/main/java/cm/aptoide/pt/dataprovider/model/v7/timeline/MinimalCard.java

```diff
@@ -1,102 +0,0 @@
-package cm.aptoide.pt.dataprovider.model.v7.timeline;
-
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import java.util.Date;
-import java.util.List;
-
-/**
- * Created by jdandrade on 11/05/2017.
- * Minimal representation of a card model.
- */
-
-public class MinimalCard {
-
-  private String cardId;
-  private Date date;
-  private SocialCardStats stats;
-  private UserSharerTimeline owner;
```

Comment:
This class should be removed since its only usages are on the Poster class that should also be removed since it is unused

Reply:
deleted